src/components/Product/MaxAI/index.tsx

```diff
@@ -0,0 +1,450 @@
+import CloudinaryImage from 'components/CloudinaryImage'
+import React, { useState } from 'react'
+import Link from 'components/Link'
+import { IconBook, IconChat, IconCode, IconGraph, IconRewindPlay, IconThoughtBubble, IconUser } from '@posthog/icons'
+import { CallToAction } from 'components/CallToAction'
+import { CustomerCard } from 'components/Products/CustomerCard'
+import { Hero } from 'components/Products/Hero'
+import { Feature } from 'components/Products/Feature'
+import { Subfeature } from 'components/Products/Subfeature'
+import { docsMenu } from '../../../navs'
+import { Marquee } from 'components/Products/Marquee'
+import TeamMembers from '../TeamMembers'
+import Questions from '../Questions'
+import CTA from 'components/Home/CTA'
+import { PairsWith } from 'components/Products/PairsWith'
+import { PairsWithItem } from 'components/Products/PairsWith/item'
+import { Question } from 'components/Products/Question'
+import { DocLinks } from 'components/Products/DocsLinks'
+import { SmoothScroll } from 'components/Products/SmoothScroll'
+import { SEO } from 'components/seo'
+import { useLayoutData } from 'components/Layout/hooks'
+import Plans from 'components/Pricing/Plans'
+import Profile from '../../Team/Profile'
+import SideModal from '../../Modal/SideModal'
+import { VsCompetitor } from 'components/Products/Competitor'
+import { VsPostHog } from 'components/Products/Competitor/VsPostHog'
+import Comparison from '../Comparison'
+import { useStaticQuery, graphql } from 'gatsby'
+import { MaxQuestionInput } from 'components/MaxQuestionInput'
+
+interface ProfileData {
+    firstName: string
+    lastName: string
+    country: string
+    companyRole: string
+    image: string
+    bio: string
+    twitter: string
+    github: string
+    linkedin: string
+    pineappleOnPizza: boolean
+    biography: string
+    isTeamLead: boolean
+    id: string
+    location: string
+}
+
+const product = {
+    slug: 'max-ai',
+    lowercase: 'Max AI',
+    capitalized: 'Max AI',
+    freeTier: 'JOETBD',
+}
+
+const team = 'Max AI'
+const teamSlug = '/teams/max-ai'
+
+const featuresPerRow = 3
+const features = [
+    {
+        title: '"How many sign-ups this week?"',
+        description: 'Get answers as big numbers, graphs, funnels, and more',
+        image: (
+            <CloudinaryImage
+                src="https://res.cloudinary.com/dmukukwp6/image/upload/big_number_e3383350cb.png"
+                width={428}
+                placeholder="none"
+            />
```

Comment:
These CloudinaryImages are pretty blurry bc the auto-scaling doesn't take the `devicePixelRatio` into account. How to fix? @smallbrownbike